pkg/api/repository.go

```diff
@@ -0,0 +1,11 @@
+package api
+
+type RepositoryRequest struct {
+	UUID                string `json:"uuid"`
+	Name                string `json:"name"`
+	URL                 string `json:"url"`                                //URL of the remote yum repository
+	DistributionVersion string `json:"distribution_version" example:"7"`   //Version to restrict client usage to
+	DistributionArch    string `json:"distribution_arch" example:"x86_64"` //Architecture to restrict client usage to
+	AccountID           string `json:"account_id"`                         //Account Id of the owner
+	OrgID               string `json:"org_id"`                             //Organization Id of the owner
```

Comment:
can we not use RepositoryItem for this?

Reply:
and maybe renaming it makes more sense.  The handler package was sorta meant to be an 'api' package, we could rename that too

Reply:
I put the 'api' package because handler and dao can't import each other. What about replacing 'RepositoryItem' and 'RepositoryRequest' with 'RepositoryAPI'?

Reply:
yeah, i guess moving all the api structs to 'api' could make sense, lets give that a try.  So i'd move RepositoryInfo & RepositoryCollectionResponse

Reply:
and we may want to move all the api structs out of handlers/api.go too.  We can leave that for another day though

Reply:
If I move RepositoryCollectionResponse, I think that means I have to move all of the api structs, since it's dependent on them. Would it make sense for api.go to live in 'api' instead of 'handler'?

Reply:
it may make sense for all the structs to live there.  I'm not sure it makes sense for the functions to?